x-pack/plugins/observability/public/pages/alerts/components/observability_actions.tsx

```diff
@@ -146,6 +152,18 @@ export function ObservabilityActions({
           {translations.alertsTable.viewAlertDetailsButtonText}
         </EuiContextMenuItem>,
       ],
+
+      ...(!!linkToAlert
```

Comment:
it's better to provide information about the types of values stored through the code. i.e. `alert.fields['kibana.alert.uuid'] ?? null;` 
So I would prefer using `Boolean(linkToAlert)` instead of `!!` as it's the more explicit conversion of type

Reply:
At a second look, I think `!!` is not needed here. I am going to remove it. Do you think we should keep Boolean(), what would be the use case?

Reply:
The goal of `!!` is to make `alert.fields['kibana.alert.uuid'] ?? null;` as boolean expression. Yes, we can remove it.